File: phonebook.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

type struct
{
    string name;
    string number;
}
person;

int main(void)
{
      person people[3];

     string name = get_string("Name: ");
     for (int i = 0; i < 3; i++)
     {
          if (strcmp(people[i].name, name) == 0)
          {
            printf("Found %s\n", people[i].number);
            return 0;
          }
     }
     printf("Not found\n");
            return 1;
}

File: phonebook.py
people = {
      "Abdo": "+1-617-495-1000",
      "Carter": "+1-617-495-1000",
      "David": "+1-949-468-2750",
}

name = input("Name: ")

for person in people:
    if person["name"] == name:
        number = person["number"]
        print(f"Found {number}")
        break
else:
    print("Not found")

